homeassistant/components/onkyo/number.py

```diff
@@ -0,0 +1,114 @@
+"""Platform for sensor integration."""
+from __future__ import annotations
```

Comment:
I find it very confusing that you are adding a new platform as part of config-flow implementation.
I think it should have been kept for a follow-up PR.